cli/internal/helm/loader_test.go

```diff
@@ -121,29 +123,11 @@ func TestConstellationServices(t *testing.T) {
 
 			result, err := engine.Render(chart, valuesToRender)
 			require.NoError(err)
-			for k, actual := range result {
-				currentFile := path.Join("testdata", tc.config.GetProvider().String(), k)
-				expected, err := os.ReadFile(currentFile)
-				// If a file does not exist, we expect the render for that path to be empty.
-				if errors.Is(err, fs.ErrNotExist) {
-					assert.YAMLEq("", actual, fmt.Sprintf("current file: %s", currentFile))
-					continue
-				}
-				require.NoError(err)
-
-				// testify has an issue where when multiple documents are contained in one YAML string,
-				// only the first document is parsed [1]. For this reason we split the YAML string
-				// into a slice of strings, each entry containing one document.
-				// [1] https://github.com/stretchr/testify/issues/1281
-				expectedSplit := strings.Split(string(expected), "\n---\n")
-				sort.Strings(expectedSplit)
-				actualSplit := strings.Split(actual, "\n---\n")
-				sort.Strings(actualSplit)
-				assert.Equal(len(expectedSplit), len(actualSplit))
-
-				for i := range expectedSplit {
-					assert.YAMLEq(expectedSplit[i], actualSplit[i], fmt.Sprintf("current file: %s", currentFile))
-				}
+			testDataPath := path.Join("testdata", tc.config.GetProvider().String(), "constellation-services")
+			err = filepath.Walk(testDataPath, compareTestdata(tc.config.GetProvider().String(), result, assert, require))
+
+			if err != nil {
+				log.Println(err)
```

Comment:
```suggestion
			require.NoError(err)
```
If you wish to print the error instead, you can use `t.Log()` or `t.Logf()`